rotkehlchen/chain/evm/decoding/curve/decoder.py

```diff
@@ -174,6 +175,16 @@ def _decode_curve_remove_events(
                     tx_log.topics[0] in REMOVE_LIQUIDITY_IMBALANCE
                 )
             ):
+                # Check if the amount matches an amount in the REMOVE_LIQUIDITY event data
+                # to prevent false matches with unrelated spend events (e.g. paraswap swaps)
+                if (token_count := REMOVE_LIQUIDITY_TOKEN_COUNTS.get(tx_log.topics[0])) is not None:  # noqa: E501
+                    token_amounts = [
+                        int.from_bytes(tx_log.data[i:i + 32]) for i in
```

Comment:
int.from_bytes() can raise TypeError right? And also can't this raise index error?

Reply:
i don't think it's needed here since the token count is deterministic from the event signatures, so we're guaranteed an integer value.